src/project_types.rs

```diff
@@ -1,17 +1,38 @@
 /// Structure to represent a Proton Project.
 /// This is what will be written to a Protonfile at the project root.
-#[derive(Debug, RustcEncodable, RustcDecodable)]
+
```

Comment:
To me it looks weird next to the `#[derive()]` statement. Move the derive inside the struct? Otherwise it does look better than having the space